gsplat/cuda/csrc/world_to_cam_bwd.cu

```diff
@@ -27,8 +27,12 @@ world_to_cam_bwd_kernel(const uint32_t C, const uint32_t N,
                         T *__restrict__ v_covars,         // [N, 3, 3]
                         T *__restrict__ v_viewmats        // [C, 4, 4]
 ) {
+
+    // For now we'll upcast float16 and bfloat16 to float32
+    using OpT = typename OpType<T>::type;
```

Comment:
This is very convenient! It basically maps the data type `T` to `OpT` that defines the precision of the calculation in this kernel.